pwiz_tools/Skyline/FileUI/PeptideSearch/ImportPeptideSearchDlg.cs

```diff
@@ -97,16 +98,25 @@ public class DDASearchPage : IFormView { }
 
         private readonly Stack<SrmDocument> _documents;
 
-        public ImportPeptideSearchDlg(SkylineWindow skylineWindow, LibraryManager libraryManager)
+        public ImportPeptideSearchDlg(SkylineWindow skylineWindow, LibraryManager libraryManager,
+            Workflow? workflowType)
         {
             SkylineWindow = skylineWindow;
             _documents = new Stack<SrmDocument>();
             SetDocument(skylineWindow.Document, null);
 
-            ImportPeptideSearch = new ImportPeptideSearch();
+            ImportPeptideSearch = new ImportPeptideSearch()
+                { IsFeatureDetection = (workflowType??Workflow.dda) == Workflow.feature_detection };

```

Comment:
You can do:
`IsFeatureDetection = workflowType == Workflow.feature_detection`
[For the comparison operators if one or both operands are null, the result is false](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types#lifted-operators).

Reply:
Good point. Tidied that up, thanks.